core/embed/rust/src/time.rs

```diff
@@ -67,6 +67,18 @@ impl Div<Duration> for Duration {
     }
 }
 
+impl From<f32> for Duration {
```

Comment:
These two conversion functions imply the `value` is in seconds and the return value below converts `Duration` to seconds as well.
I think this is error-prone. Being explicit would be better here. Do we need it?

Reply:
okay lets be explicit https://github.com/trezor/trezor-firmware/pull/3776/commits/834ede0d3f37cd4d9d10e1d5a89626cef78385b6